aip/auth/4110.md

```diff
@@ -118,9 +118,7 @@ digraph d_front_back {
 1. **Check environment variables**
   1. Check GOOGLE_APPLICATION_CREDENTIALS
       1. If set, go to step (2.2)
-  1. Check GOOGLE_API_KEY
-      1. If set, use the provided API key _[END]_
-  1. If neither of above is set, go to step (2)
+      1. If not set, go to step (2)
```

Comment:
You will need to remove the API_KEY mention at L67 as well.

Reply:
Done, good call!